corehq/apps/builds/templates/builds/edit_menu.html

```diff
@@ -132,47 +132,85 @@ <h3 class="panel-title">
             {% trans "Version*" %}
           </label>
           <div class="col-sm-8">
-            <input id="import_build_version" type="text" name="version" class="form-control" />
-          </div>
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-3" for="import_build_build_number">
-            {% trans "Build Number*" %}
-          </label>
-          <div class="col-sm-8">
-            <input id="import_build_build_number" type="text" name="build_number" class="form-control" />
+            <input id="import_build_version" type="text" name="version" class="form-control"/>
             <ol class="help-block">
-              <li><a href="http://jenkins.dimagi.com/view/CommCare%20Mobile/" target="_blank">{% trans "Browse builds" %}</a></li>
               <li>
-                {% blocktrans %}
-                  Click on the appropriate commcare-android-2.X link
-                {% endblocktrans %}
+                <a href="https://github.com/dimagi/commcare-android/releases" target="_blank">
+                  {% trans "Browse versions" %}
+                </a>
               </li>
-              <li>{% trans "Click on the most recent successful build in the table at left" %}</li>
-              <li>{% trans "Grab the version, a six-digit number, from the URL" %}</li>
-            </ol>
-          </div>
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-3" for="import_build_name">
-            {% trans "Source for artifacts.zip (J2ME only)" %}
-          </label>
-          <div class="col-sm-8">
-            <input id="import_build_name" type="text" name="source" class="form-control" />
-            <ol class="help-block">
-              <li><a href="http://jenkins.dimagi.com/view/CommCare%20Mobile/" target="_blank">{% trans "Browse builds" %}</a></li>
+              <li>{% trans "Paste the latest version in the box above in the format x.y.z" %}</li>
+              <li>{% trans "Click Import Build" %}</li>
               <li>
-                {% blocktrans %}
-                  Click on the appropriate commcare-core-2.X link
-                {% endblocktrans %}
+                {% trans "You can alternatively follow" %}
+                <a
+                  href="https://github.com/dimagi/commcare-cloud/blob/master/docs/setup/new_environment.md#add-a-new-commcare-build"
+                  target="_blank">
+                  {% trans "these steps" %}
+                </a>
+                {% trans "to automatically get the latest version" %}
```

Comment:
You should use `blocktrans` here instead, so these strings can be translated as a whole, otherwise the translators won't be able to rephrase in a way that makes sense in their language.  Plus they see each individual string in isolation without much context.

Also, since the steps described are simply running a command, I'd prefer just pasting that command in here, even if it means the information lives in more than one place.

Reply:
So, something like:
"Alternatively, you can run <command> on the deploy server. See <link> for reference" ?

Reply:
I mean something like.
> You can automatically fetch the latest build from github using the following management command
> `./manage.py add_commcare_build --latest`

Reply:
Cool. I see now my previous comment left out half of my sentence in the quotes. I think I might have used odd formatting or something. Odd...